pawn/scripting/include/smjansson.inc

```diff
@@ -889,6 +889,11 @@ stock Handle json_string_format_ex(int tmpBufferLength, const char[] sFormat, an
 stock bool json_array_get_bool(Handle hArray, int iIndex) {
 	Handle hElement = json_array_get(hArray, iIndex);
 
+	if (hElement == INVALID_HANDLE) {
```

Comment:
invalid_handle is an enum that points to zero so deleting will do nothing
- https://github.com/alliedmodders/sourcemod/blob/d42a8c7c1e3f1da1fc01baae6cdeda71920c3cb8/plugins/include/handles.inc#L43